cleanlab/object_detection/filter.py

```diff
@@ -181,10 +196,10 @@ def _find_label_issues(
 
 
 def _find_label_issues_per_box(
-    scores_per_box: List[np.ndarray], threshold: float
+    scores_per_box: List[np.ndarray], thr_classes, threshold_factor=1.0
```

Comment:
inconsistent variable naming, `thr_classes` vs `threshold_factor`. I say we just use `threshold_classes` aka write out `threshold` instead of `thr` in variables (not sure if this applies to other vars as well)